examples/footer/index.html

```diff
@@ -8,59 +8,85 @@
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal.html" title="Disclaimer / Registrations / Copyright Statement">Disclaimer / Registrations / Copyright Statement</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/linking-policy.html" title="Linking Policy">Linking Policy</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-privacy-practices.html" title="Notice of Privacy Practices (HIPAA)">Notice of Privacy Practices (HIPAA)</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-non-discrimination.html" title="Notice of Non-Discrimination">Notice of Non-Discrimination</a>
-          </li>
-        </ul>
-        <ul class="social-media">
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/stjuderesearch" class="fab fa-facebook-f">
-              <span class="sr-only">Facebook</span>
+    <footer>
+      <div class="container">
+        <div class="row">
+          <div class="col-6 col-lg-3 col-md-4 offset-xl-0 offset-lg-5 offset-md-4 offset-md-3 offset-3 footer-logo">
+            <a class="navbar-brand" href="https://stjude.cloud">
+              <span class="logo"></span>
             </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/StJudeResearch" class="fab fa-twitter">
-              <span class="sr-only">Twitter</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.instagram.com/stjuderesearch/" class="fab fa-instagram">
-              <span class="sr-only">Instagram</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/company/st-jude-childrens-research-hospital/" class="fab fa-linkedin">
-              <span class="sr-only">LinkedIn</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/user/MyStJude" class="fab fa-youtube">
-              <span class="sr-only">YouTube</span>
-            </a>
-          </li>
-        </ul>
+            <a class="sjc-title" href="https://stjude.cloud">St. Jude Cloud</a>
+          </div>
+          <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-about">
+            <h6>About St. Jude</h6>
+            <ul>
+              <li>
+                <a href="https://www.stjude.org">Home</a>
+              </li>
+              <li>
+                <a href="https://www.stjude.org/about-st-jude.html">About</a>
+              </li>
+              <li>
+                <a href="https://stjude.org/contact-us.html">Contact</a>
```

Comment:
Question to the group: is it more important that we have our own contact information at the bottom of the page? Many people will just naturally scroll here and click "Contact", which will point them to the general St. Jude contact page.

@jamesmadson, I think it may be worth swapping this out (I can't imagine many people would actually intend to contact the link described here).

Reply:
@claymcleod are you suggesting changing the link to [this one (which is currently in the footer)](https://hospital.stjude.org/apps/forms/fb/st-jude-cloud-contact/) or creating a new one that's hosted on St. Jude Cloud rather than the hospital's website? 

At minimum, @jamesmadson, I think the link should be swapped out for the one I linked to above, but from a UX perspective, I'm in favor of a contact us page on SJC depending on what our options are. Our own page would also be an opportunity to present a link to the docs (which is no longer in the footer).